static/app/components/feedback/feedbackOnboarding/sidebar.tsx

```diff
@@ -43,12 +43,15 @@ function FeedbackOnboardingSidebar(props: CommonSidebarProps) {
   const isActive = currentPanel === SidebarPanelKey.FEEDBACK_ONBOARDING;
   const hasProjectAccess = organization.access.includes('project:read');
 
-  const {projects, currentProject, setCurrentProject} = useCurrentProjectState({
+  const {allProjects, currentProject, setCurrentProject} = useCurrentProjectState({
```

Comment:
`projects` --> `allProjects` throughout this file is just a variable name change; no functionality change here